src/redux/modules/posts.js

```diff
@@ -63,11 +90,24 @@ const convertPostsToPlain = posts => {
     };
 }
 
+const convertSinglePostToPlain = post => {
+    const plainPost = {...post, author: post.author.id};
+    const author = {...post.author};
+    return {
+        post: plainPost,
+        author: author
```

Comment:
If directly open a post detail page, it's easy to understand why to load the author here.
If open a post detail from post list view, here also retrieve the author, and just refresh the corresponding user in the user list that was loaded when getting post list data.